### DISCUSSION TITLE
docker pull error

### DISCUSSION THREAD

**Discussion by ReneXB12X** [blakeblackshear/frigate#13569] [General]
rene@boot-sr:~ $ docker pull ghcr.io/blakeblackshear/frigate:cache-arm64
Error response from daemon: Get "https://ghcr.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
rene@boot-sr:~ $

**Comment by hawkeye217** [comment]
You should be pulling the correct image from the releases page: https://github.com/blakeblackshear/frigate/releases

**Reply by ReneXB12X** [reply]
rene@boot-sr:~ $ docker pull ghcr.io/blakeblackshear/frigate:0.14.1-standard-arm64
Error response from daemon: Get "https://ghcr.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
rene@boot-sr:~ $

**Reply by hawkeye217** [reply]
Perhaps GHCR is temporarily down or something with the network isn’t working on your end?

**Reply by ReneXB12X** [reply]
I can reach this site:
https://github.com/blakeblackshear/frigate/pkgs/container/frigate
but none of the  tagged image versions!

**Reply by NickM-27** [reply]
works fine for me 

<img width="418" alt="Screenshot 2024-09-05 at 11 15 18 AM" src="https://github.com/user-attachments/assets/1113c180-1463-4780-9bb3-e3b89316cd10">
